LEAF_Request_Portal/js/vue_conditions_editor/LEAF_conditions_editor.js

```diff
@@ -707,9 +707,8 @@ const ConditionsEditor = Vue.createApp({
      * @returns {Array} of condition objects
      */
     savedConditions() {
-      return this.childIndicator.conditions
-        ? JSON.parse(this.childIndicator.conditions)
-        : [];
+      return typeof this.childIndicator.conditions === 'string' && this.childIndicator.conditions[0] === '[' ?

```

Comment:
```suggestion
      return typeof this.childIndicator.conditions === 'string' && this.childIndicator.startsWith('[') ?

```

Reply:
I'll note this for when I do the next round of refresh updates.  I did double-check to make sure that [0] wouldn't break it here in the case of an empty string, so both essentially do the same thing.  We don't need to worry about IE any more though, so this choice is a bit neater.